### PR TITLE
[WIP] ZEPPELIN-1761. CI: ZeppelinSparkClusterTest .pySpark* fail the build

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?
Add more logging as I could not reproduce it locally. 

### What type of PR is it?
[ Improvement ]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-1761

### How should this be tested?
Outline the steps to test the PR here.

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by zjffdu** [comment]
This PR is not ready for review, I just add more logging to reproduce it in CI.

**Comment by felixcheung** [comment]
could you mark the title with [WIP]?
let me know when this is ready!

**Comment by felixcheung** [comment]
where are we on this?

**Comment by zjffdu** [comment]
Didn't see the error recently, let me try again.

**Comment by AlexanderShoshin** [comment]
I had similar issue a week ago:
log - https://api.travis-ci.org/jobs/201058125/log.txt?deansi=true
config - https://travis-ci.org/apache/zeppelin/jobs/201058125/config
